src/.eslintrc.js

```diff
@@ -1,7 +1,3 @@
-'use strict';
-
 module.exports = {
-	"env": {
-		"browser": true
-	}
-}
+	"extends": "origami-component"
```

Comment:
If all JavaScript files in the. `src` folder are for the client-side/browser, we should move to using `export`/`import` and not `module.exports`/`require` :-)

Reply:
👍 I cleared up the remaining `module.exports`